usaspending_api/api_docs/markdown/data_dictionary.md

```diff
@@ -0,0 +1,750 @@
+| total_outlay | Float | The total amount of money paid out for this award |
+| awarding_agency | Relation | The awarding agency for the award |
+| funding_agency | Relation | The funding agency for the award |
+| date_signed | Date | The date the award was signed |
+| recipient | Relation | The recipient of the award |
+| description | String | A description of the award |
+| period_of_performance_start_date | Date | The start date for the period of performance |
+| period_of_performance_current_end_date | Date | The current, not original, period of performance end date |
+| place_of_performance | Relation | The principal place of business, where the majority of the work is performed. For example, in a manufacturing contract, this would be the main plant where items are produced. |
+| potential_total_value_of_award | Float | The sum of the potential_value_of_award from associated transactions |
+| last_modified_date | Date | The date this award was last modified |
+| certified_date | Date | The date this record was certified |
+| create_date | Datetime | The date this record was created in the API |
+| update_date | Datetime | The last time this record was updated in the API |
+| latest_submission | Relation | The submission attribute object that created this award |
+| latest_transaction | Relation | The latest transaction by action_date associated with this award |
+
+#### Transaction Endpoints <a name="transaction"></a>
+
+Transaction data is returned as a combination of the three following models:
+This data is represented internally as the model: `Transaction`
+
+
+| Field | Type | Description |
+| ----- | ----- | ----- |
+| latest_for_award | Relation | Reverse look-up for relation from Award::latest_transaction |
+| contract_data | Relation | Reverse look-up for relation from TransactionContract::transaction |
+| assistance_data | Relation | Reverse look-up for relation from TransactionAssistance::transaction |
+| id | Integer | Internal primary key. Guaranteed to be unique. |
+| data_source | String | The source of this entry, either Data Broker (DBR) or USASpending (USA) |
+| award | Relation | The award which this transaction is contained in |
+| usaspending_unique_transaction_id | String | If this record is legacy USASpending data, this is the unique transaction identifier from that system |
+| submission | Relation | The submission which created this record |
+| type | String | The type for this transaction. For example, A, B, C, D |
+| type_description | String | The plain text description of the transaction type |
+| period_of_performance_start_date | Date | The period of performance start date |
+| period_of_performance_current_end_date | Date | The current end date of the period of performance |
+| action_date | Date | The date this transaction was actioned |
+| action_type | String | The type of transaction. For example, A, B, C, D |
```

Comment:
I'm skimming through, but for fields with code responses, I think we should either document or link to an external document (DAIMS?) of all the possible response codes and what they mean.